.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- Please use the search function to check if the desired function has already been requested by someone else -->
+<!-- If you want to report a bug, please use "Bug report" instead -->
+
+**Is your feature request related to a problem? Please describe.**
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+**Describe the solution you'd like**
+<!-- A clear and concise description of what you want to happen. -->
+
+**Describe alternatives you've considered**
+<!-- A clear and concise description of any alternative solutions or features you've considered. -->
+
+**Additional context**
```

Comment:
"If you know the part of the code that need to be enhanced, please link it here"

Reply:
Most won't know the code and if people know the code they automatically will link it, that's what already happened in the past without any template. This additional sentence is just more reading for the normal user.

Reply:
Agree.